ramls/instance.json

```diff
@@ -403,6 +403,12 @@
         "$ref": "uuid.json"
       }
     },
+    "isBoundWith": {
+      "description": "Indicates if this Instance is included in a bound-with",
+      "type": "boolean",
+      "default": false,
+      "readonly": true
+    },
```

Comment:
Given that an instance is an abstract description of a bibliographic entity, how can an instance be bound with anything?